### ISSUE TITLE
Doesn't work with RPi3B+

### ISSUE THREAD

**Issue by SuperNose** [RasPlex/rasplex-installer#50]
The boot doesn't even start - only the color rainbow is displayed.

Trying with Raspberry PI 3B+. Official Raspbian etc. works as expected, so the HW is fine.

Thank you

**Comment by issacdickinson** [comment]
Same here

**Comment by apfelnz** [comment]
Same problem

**Comment by SIBUK** [comment]
I also have the same issue

**Comment by Risca** [comment]
Might be a problem with openpht not supporting the hw. Have anyone tried flashing the image manually?

**Comment by MDanihy** [comment]
Same here, doesn't work.  Funny how the website says it does, but the installer doesn't even support it either, just the RPi and RPi2.

**Comment by Risca** [comment]
I've successfully used it for RPi 3 before. I wonder what has changed 🤔

**Comment by MDanihy** [comment]
Used the image and Apple Pi Baker, same thing, the image no longer works.  Will try the manual DD route but I have a feeling the image is borked for Pi3, and with the Pi4 out, I assume this will not work either.

**Comment by Risca** [comment]
Please note that the images that openpht host are gzipped and needs to be unpacked before using dd

**Comment by MDanihy** [comment]
Yup

**Comment by MDanihy** [comment]
dd instructions are wrong as well, dd: bs: illegal numeric value.  This project needs more testers...

**Comment by Risca** [comment]
Indeed! Unfortunately, OpenPHT lost steam when one of the best testers they had unfortunately passed away about a year ago
EDIT: 3 years ago. Wow, time really flies by sometimes

**Comment by MDanihy** [comment]
Well, the developer says he will be working on a RPi4 version, but he does state he broke the Pi3B+.  Seems I have to search for a beta version somewhere.  I'll probably just scrap the idea and just run everything through the browser.

**Comment by dalehamel** [comment]
Locking this, as the status has been well covered and issues aren’t for venting. 

I know it’s frustrating to have to wait, and I’m sorry for that.
